inc/config.class.php

```diff
@@ -551,68 +553,68 @@ public static function install(Migration $migration) {
-            $migration->dropField($table, "default_itilcategories_id");
-
-            $migration->addField($table, "hide_inactive_budgets", "bool");
-            $migration->addField($table, "rename_documents", "bool");
-
-            //0.85+1.2
-            $migration->addField($table, "transmit_budget_change", "bool");
-
-            $migration->migrationOneTable($table);
-            if ($templateID) {
-               $config->update(array('id' => 1, 'tickettemplates_id_delivery' => $templateID));
-            }
+      } else {
+         //Upgrade
+         $migration->displayMessage("Upgrading $table");
+
+         //1.2.0
+         $migration->renameTable("glpi_plugin_order_config", $table);
+
+         if (!countElementsInTable("glpi_plugin_order_configs")) {
+            $query = "INSERT INTO `glpi_plugin_order_configs`(`id`,`use_validation`,`default_taxes`) VALUES (1,0,0);";
+            $DB->query($query) or die($DB->error());
+         }
+
+         $migration->changeField($table, "ID", "id", "int(11) NOT NULL auto_increment");
+
+         //1.3.0
+         $migration->addField($table, "generate_assets", "tinyint(1) NOT NULL default '0'");
+         $migration->addField($table, "generated_name", "varchar(255) collate utf8_unicode_ci default NULL");
+         $migration->addField($table, "generated_serial", "varchar(255) collate utf8_unicode_ci default NULL");
+         $migration->addField($table, "generated_otherserial", "varchar(255) collate utf8_unicode_ci default NULL");
+         $migration->addField($table, "default_asset_entities_id", "int(11) NOT NULL default '0'");
+         $migration->addField($table, "default_asset_states_id", "int(11) NOT NULL default '0'");
+         $migration->addField($table, "generated_title", "varchar(255) collate utf8_unicode_ci default NULL");
+         $migration->addField($table, "generated_content", "text collate utf8_unicode_ci");
+         $migration->addField($table, "default_ticketcategories_id", "int(11) NOT NULL default '0'");
+         $migration->addField($table, "use_supplier_satisfaction", "tinyint(1) NOT NULL default '0'");
+         $migration->addField($table, "generate_order_pdf", "tinyint(1) NOT NULL default '0'");
+         $migration->addField($table, "use_supplier_informations", "tinyint(1) NOT NULL default '1'");
+         $migration->addField($table, "shoudbedelivered_color", "char(20) collate utf8_unicode_ci default '#ff5555'");
+         $migration->addField($table, "copy_documents", "tinyint(1) NOT NULL DEFAULT '0'");
+         $migration->addField($table, "documentcategories_id", "integer");
+         $migration->addField($table, "groups_id_author", "integer");
+         $migration->addField($table, "groups_id_recipient", "integer");
+         $migration->addField($table, "users_id_recipient", "integer");
+
+         $migration->changeField($table, "default_ticketcategories_id",
+                              "default_itilcategories_id", "integer");
```

Comment:
This part of the indentation is bad, we need to align the "default_ with the above $table